Sage/carbon
# Fixed missing SCSS import for Page component

# Description
Carbon version 5.0.0 introduced a small regression by removing SCSS import from package.json but not adding it to `Page` component itself, making it miss few CSS rules.

# TODO

# Screenshots / Gifs

# Testing Instructions
Confirm that the padding required in Journals View is back. This should be enough to ensure that the Page SCSS is now present.
